arm64-assembly-optimization.md

```diff
@@ -0,0 +1,95 @@
+# Assembly Optimization Guide for Graviton Arm64 Processors
+
+## Introduction
+
+This document is a reference for software developers who want to optimize code
+for Graviton Arm64 processors at the assembly level. Some of the documented
```

Comment:
Arm64 is capitalized here, but then not near the bottom (line 85). Maybe it's intentional given the usage after Graviton here, but just pointing it out.